<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TPFourClass</class>
 <widget class="QMainWindow" name="TPFourClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TPFour</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QSlider" name="slideTemperature">
    <property name="geometry">
     <rect>
      <x>79</x>
      <y>160</y>
      <width>391</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="btnAllumer">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>60</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Allumer</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnEteindre">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>60</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Eteindre</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblPuissance">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>240</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Puissance :</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblnbPuissance">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>240</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="lblnbTemperature">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>240</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="lblTemperature">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>240</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Temperature :</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="TPFour.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>btnAllumer</sender>
   <signal>clicked()</signal>
   <receiver>TPFourClass</receiver>
   <slot>allumerFour()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>145</x>
     <y>113</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnEteindre</sender>
   <signal>clicked()</signal>
   <receiver>TPFourClass</receiver>
   <slot>eteindreFour()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>405</x>
     <y>113</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>slideTemperature</sender>
   <signal>valueChanged(int)</signal>
   <receiver>TPFourClass</receiver>
   <slot>sliderTemp()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>274</x>
     <y>203</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>allumerFour()</slot>
  <slot>eteindreFour()</slot>
  <slot>sliderTemp()</slot>
 </slots>
</ui>
